test/server/http/admin_test.cc

```diff
@@ -776,16 +780,33 @@ TEST_P(AdminInstanceTest, ClustersJson) {
        "port_value": 80
       }
      },
-     "stats": {
-      "test_counter": {
+     "stats": [
+       {
+       "name": "arest_counter",
+       "value": "5",
+       "type": "COUNTER"
+       },
+       {
+       "name": "rest_counter",
```

Comment:
How is the ordering of the named fields determined in the json output?

Reply:
Sorry. I probably did not understand your question fully but this how it will look like in the output, 
  "stats": [
      {
       "type": "COUNTER",
       "value": "0",
       "name": "cx_connect_fail"
      },
      {
       "type": "COUNTER",
       "value": "0",
       "name": "cx_total"
      },
      {
       "type": "COUNTER",
       "value": "0",
       "name": "rq_error"
      },
      {
       "type": "COUNTER",
       "value": "0",
       "name": "rq_success"
      },
      {
       "type": "COUNTER",
       "value": "0",
       "name": "rq_timeout"
      },
      {
       "type": "COUNTER",
       "value": "0",
       "name": "rq_total"
      },
      {
       "type": "GAUGE",
       "value": "0",
       "name": "cx_active"
      },
      {
       "type": "GAUGE",
       "value": "0",
       "name": "rq_active"
      }
     ],

Reply:
I guess what I'm asking is - are we depending on some undocumented proto->json field ordering scheme for this to work?  Or is the ordering here well defined and expected to be reasonably stable?

Reply:
I see. The test doesn't compare the string json, so we're not dependent on it. I assumed that you were doing a direct string comparison. As long as we don't care about the individual field ordering being stable over time, this should be totally fine. Sorry for the hubbub over nothing :)

Reply:
No problem :-)